integration-tests/plugins/__tests__/inventory/order/order.js

```diff
@@ -0,0 +1,267 @@
+const path = require("path")
+
+const { bootstrapApp } = require("../../../../helpers/bootstrap-app")
+const { initDb, useDb } = require("../../../../helpers/use-db")
+const { setPort, useApi } = require("../../../../helpers/use-api")
+
+const adminSeeder = require("../../../helpers/admin-seeder")
+const cartSeeder = require("../../../helpers/cart-seeder")
+const { simpleProductFactory } = require("../../../../api/factories")
+const { simpleSalesChannelFactory } = require("../../../../api/factories")
+const {
+  simpleOrderFactory,
+  simpleRegionFactory,
+} = require("../../../factories")
+
+jest.setTimeout(30000)
+
+const adminHeaders = { headers: { Authorization: "Bearer test_token" } }
+
+describe("/store/carts", () => {
+  let express
+  let appContainer
+  let dbConnection
+
+  const doAfterEach = async () => {
+    const db = useDb()
+    return await db.teardown()
+  }
+
+  beforeAll(async () => {
+    const cwd = path.resolve(path.join(__dirname, "..", "..", ".."))
+    dbConnection = await initDb({ cwd })
+    const { container, app, port } = await bootstrapApp({ cwd })
+    appContainer = container
+
+    setPort(port)
+    express = app.listen(port, (err) => {
+      process.send(port)
+    })
+  })
+
+  afterAll(async () => {
+    const db = useDb()
+    await db.shutdown()
+    express.close()
+  })
+
+  afterEach(async () => {
+    jest.clearAllMocks()
+    const db = useDb()
+    return await db.teardown()
+  })
+
+  describe("POST /store/carts/:id", () => {
+    let order
+    let locationId
+    let invItemId
+    let variantId
+    let prodVarInventoryService
+
+    beforeEach(async () => {
+      const api = useApi()
+
+      prodVarInventoryService = appContainer.resolve(
+        "productVariantInventoryService"
+      )
+      const inventoryService = appContainer.resolve("inventoryService")
+      const stockLocationService = appContainer.resolve("stockLocationService")
+      const salesChannelLocationService = appContainer.resolve(
+        "salesChannelLocationService"
+      )
+
+      const r = await simpleRegionFactory(dbConnection, {})
+      await simpleSalesChannelFactory(dbConnection, {
+        id: "test-channel",
+        is_default: true,
+      })
+
+      await adminSeeder(dbConnection)
+
+      const product = await simpleProductFactory(dbConnection, {
+        id: "product1",
+        sales_channels: [{ id: "test-channel" }],
+      })
+      variantId = product.variants[0].id
+
+      const sl = await stockLocationService.create({ name: "test-location" })
+
+      locationId = sl.id
+
+      await salesChannelLocationService.associateLocation(
+        "test-channel",
+        locationId
+      )
+
+      const invItem = await inventoryService.createInventoryItem({
+        sku: "test-sku",
+      })
+      invItemId = invItem.id
+
+      await prodVarInventoryService.attachInventoryItem(variantId, invItem.id)
```

Comment:
Question: Deos that mean that the SKU that is used to create the inventory item can be different than the variant SKU? are they meaning different thing?

Reply:
yes, you will still have access to all the variants stock related fields, however they wont impact behavior if you use the inventory module.